.github/actions/install-galaxy-deps/action.yml

```diff
@@ -22,6 +22,11 @@ runs:
         key: ansible-galaxy-v${{ inputs.cache-version }}-${{ inputs.cache-name }}-${{ hashFiles('requirements.yml') }}
 
     - name: Fetch dependencies from Ansible Galaxy
-      run: ansible-galaxy install -r requirements.yml --force
+      run: |
+        if [ -f Pipfile ]; then
```

Comment:
better check for `~/.cache/pipenv` existance

you can avoid duplicating the command either by:
* having a var which can be empty or contains `pipenv run`
* having a var for the entire ansible-galaxy command and args